### PR TITLE
Update readme.txt

### PR DESCRIPTION
**Pull request by ChrisJon02**
The link of latest release of Sakai is broken. Fixed that link

### CODE REVIEW

File: docs/readme.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ The official website for Sakai is:
 	
 The latest release of Sakai can always be found at:
 
-	https://www.sakailms.org/release
+	http://source.sakaiproject.org/release/22.2/
```
Comment (jesusmmp) [inline docs/readme.txt:10]: http://source.sakaiproject.org/release/

**Review by jesusmmp — APPROVED** [review]

File: docs/readme.txt (hunk 1 above)
Comment (ChrisJon02) [reply docs/readme.txt]: Updated the changes.

Comment (ChrisJon02) [reply docs/readme.txt]: Will you please merge the PR

Comment (jesusmmp) [reply docs/readme.txt]: Thanks!. I don't have permissions but I'll get someone else to do it.

Comment (ChrisJon02) [reply docs/readme.txt]: Thank you!

**Comment by ottenhoff** [comment]
Thanks for the fix

**Pull request merged**